### PR TITLE
Homework day17 improve tests

### PR DESCRIPTION
**Pull request by lyaleko**
Please look at this code

### CODE REVIEW

File: src/test/java/pages/booking/BookingHomePage.java [hunk 1]
```diff
@@ -75,19 +76,29 @@ public void setNumberOfRooms(int number) {
...
+
+    public String getTextAccountTitle() {
+        return String.format(driver.findElement(By.xpath(ACCOUNT_TITLE)).getText());
+    }
 
+    public void closePopup(){
+        driver.findElement(By.xpath(CLOSE_POPUP_BUTTON)).click();
     }
 
+    public BookingHomePage(WebDriver driver) {
```
Comment (vpetrushka) [inline src/test/java/pages/booking/BookingHomePage.java:101]: just create Driver object

File: src/test/java/pages/trashmail/TrashMailGridPage.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    private String getName() {
+        return driver.findElement(By.xpath(EMAIL_NAME_INPUT)).getText();
+    }
+
+    public String getEmailText() {
+        String name = getName();
+        return getName() + domainInput;
+    }
+
+    public TrashMailGridPage(WebDriver webDriver) {
```
Comment (vpetrushka) [inline src/test/java/pages/trashmail/TrashMailGridPage.java:30]: Create instance of Driver

File: src/test/java/pages/trashmail/TrashMailHomePage.java [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+        driver.navigate().to(TRASH_MAIL_SITE);
+    }
+
+    public void fillFormToLogin(String email, String password){
+        driver.findElement(By.xpath(LOGIN_INPUT)).sendKeys(email);
+        driver.findElement(By.xpath(PASSWORD_INPUT)).sendKeys(password);
+        driver.findElement(By.xpath(LOGIN_BUTTON)).click();
+    }
+
+    public TrashMailHomePage(WebDriver webDriver) {
```
Comment (vpetrushka) [inline src/test/java/pages/trashmail/TrashMailHomePage.java:28]: create instance of driver

File: src/test/java/pages/booking/BookingSignInPage.java [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+public class BookingSignInPage {
+
+    WebDriver driver = BasicDriver.getWebDriver();
+
+    private String EMAIL_INPUT = "//input[@type='email']";
+    private String PASSWORD_FIRST_INPUT = "//div[@class='password-input-container bui-spacer--large'][1]//input[@type='password']";
+    private String PASSWORD_CONFIRMED_INPUT = "//div[@class='password-input-container bui-spacer--large'][2]//input[@type='password']";
+    public String emailText;
+    public String emailTextTestUser = "marina.tester@trashmail.se";
+    public String passwordText = "555666777888Mm"; //TODO SHOULD KNOW IS PUBLIC REQUIRED
```
Comment (vpetrushka) [inline src/test/java/pages/booking/BookingSignInPage.java:16]: create password creater

File: src/test/java/pages/booking/BookingSignInPage.java [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+    public void fillPasswordLoginForm(String passwordText){
+        driver.findElement(By.xpath(PASSWORD_FIRST_INPUT)).sendKeys(passwordText);
+        driver.findElement(By.xpath(SUBMIT_BUTTON)).click();
+    }
+
+    public void confirmRegistrationNotBot(){
+        driver.findElement(By.xpath(BUTTON_AGAINST_ROBOT)).click();
+    }
+
+    public BookingSignInPage(WebDriver webDriver) {
```
Comment (vpetrushka) [inline src/test/java/pages/booking/BookingSignInPage.java:43]: create instance of webdriver

File: src/test/java/pages/booking/BookingHomePage.java (hunk 1 above)
Comment (lyaleko) [reply src/test/java/pages/booking/BookingHomePage.java:101]: Error from java: constructor BookingHomePage in class pages.booking.BookingHomePage cannot be applied to given types;

**Pull request merged**